adam-core/src/test/scala/org/bdgenomics/adam/rdd/SortedGenomicRDDSuite.scala

```diff
@@ -313,6 +314,7 @@ class SortedGenomicRDDSuite extends SparkFunSuite {
     assert(h.count == i.count)
   }
 
+  /*
```

Comment:
This is the one that I care about. We need to verify that we read things in with the same order and partition schema as it was written.